python/felis/cli.py

```diff
@@ -198,6 +198,11 @@ def create(
 @click.option("--dry-run", "-D", is_flag=True, help="Execute dry run only. Does not insert any data.")
 @click.option("--echo", "-e", is_flag=True, help="Print out the generated insert statements to stdout")
 @click.option("--output-file", "-o", type=click.Path(), help="Write SQL commands to a file")
+@click.option(
+    "--force-unbounded-arraysize",
+    is_flag=True,
+    help="Force unbounded array size for all variable length string columns, e.g., ``votable:arraysize: *``",
```

Comment:
How about "use unbounded arraysize by default for all variable-length string columns",

and a comment in the code that explains why this option is even there at all.